### DISCUSSION TITLE
Add example for a more advanced table for sorting, filtering, selecting, pagination

### DISCUSSION THREAD

**Discussion by Rednas83** [saadeghi/daisyui#2009] [Ideas / feature requests]
Can you also provide examples for sorting, filtering, pagination on tables? 

**Something like:**
![image](https://github.com/saadeghi/daisyui/assets/25219229/b15eabde-4da1-414d-8b61-7b17613d5e54)

**Comment by saadeghi** [comment]
There are pagination examples on the site.

You can use buttons, icons or select for sorting and filtering but daisyUI can not provide functionality because it's just CSS (just visualization) You would need to write don't JS for the functionality.

**Comment by Rednas83** [comment]
Adding the script on a later moment is no problem.  

I am just looking for an example with a more advanced table

Something like:
-  Arrow icons in the first row (for toggling sorting up/down)
-  Text input on probably second row (for filtering rows)
-  Pagination on last row (for selecting different row sets)
-  Checkboxes on first column (for selecting rows)
-  Edit icon on last column (for editing rows)
